src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(num1, num2) {
+      return num1 + num2;
+    },
+
+    subtract(num1, num2) {
+      return num1 - num2;
+    },
+
+    divide(num1, num2) {
+      return num1 / num2;
+    },
+
+    multiply(num1, num2) {
+      return num1 * num2;
+    },
```

Comment:
```suggestion
    add(currentValue, number) {
      return currentValue+ number;
    },

    subtract(num1, num2) {
      return num1 - num2;
    },

    divide(num1, num2) {
      return num1 / num2;
    },

    multiply(num1, num2) {
      return num1 * num2;
    },
```

Naming of arguments and variables is important, num1 and num2 are not informative enough and not quite correct, it should be currentValue, and number